x-pack/platform/plugins/shared/streams_app/public/components/data_management/stream_detail_enrichment/page_content.tsx

```diff
@@ -195,15 +225,95 @@ const ProcessorsEditor = React.memo(() => {
         )}
         <AddProcessorPanel />
       </EuiPanel>
+      <EuiPanel paddingSize="m" hasShadow={false} grow={false}>
+        {!isEmpty(errors.ignoredFields) && (
+          <EuiPanel paddingSize="s" hasShadow={false} grow={false} color="danger">
+            <EuiAccordion
+              id="ignored-fields-failures-accordion"
+              initialIsOpen
+              buttonContent={
+                <strong>
+                  {i18n.translate(
+                    'xpack.streams.streamDetailView.managementTab.enrichment.ignoredFieldsFailure.title',
+                    { defaultMessage: 'Some fields were ignored during the simulation.' }
+                  )}
+                </strong>
+              }
+            >
+              <EuiText component="p" size="s">
+                <p>
+                  {i18n.translate(
+                    'xpack.streams.streamDetailView.managementTab.enrichment.ignoredFieldsFailure.description',
+                    {
+                      defaultMessage:
+                        'Some fields in these documents were ignored during the ingestion simulation. Review the fields’ mapping limits.',
+                    }
+                  )}
+                </p>
+                <p>
+                  <FormattedMessage
+                    id="xpack.streams.streamDetailView.managementTab.enrichment.ignoredFieldsFailure.fieldsList"
+                    defaultMessage="The ignored fields are: {fields}"
+                    values={{
+                      fields: (
+                        <EuiFlexGroup
+                          gutterSize="s"
+                          css={css`
+                            margin-top: ${euiTheme.size.s};
+                          `}
+                        >
+                          {errors.ignoredFields.map((field) => (
+                            <EuiCode>{field}</EuiCode>
+                          ))}
+                        </EuiFlexGroup>
+                      ),
+                    }}
+                  />
+                </p>
+              </EuiText>
+            </EuiAccordion>
+          </EuiPanel>
+        )}
+        {!isEmpty(errors.mappingFailures) && (
+          <EuiPanel paddingSize="s" hasShadow={false} grow={false} color="danger">
+            <EuiAccordion
+              id="mapping-failures-accordion"
+              initialIsOpen
+              buttonContent={i18n.translate(
+                'xpack.streams.streamDetailView.managementTab.enrichment.fieldMappingsFailure.title',
+                {
+                  defaultMessage: 'Field conflicts during simulation',
+                }
+              )}
+            >
+              <EuiText size="s">
+                <p>
+                  <FormattedMessage
+                    id="xpack.streams.streamDetailView.managementTab.enrichment.fieldMappingsFailure.fieldsList"
+                    defaultMessage="These are some mapping failures occurred during the simulation:"
```

Comment:
Could we tweak the wording slightly: "These are some mapping failures **that** occurred during the simulation"

Reply:
Addressed with [refactor(streams): update copies](https://github.com/elastic/kibana/pull/216877/commits/22e2e7c902e9a0278e681a236bbc683bc04bd39d)